kolibri/core/assets/src/views/PaginatedListContainer.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+
+  <div>
+    <KGrid>
+      <KGridItem sizes="3, 3, 3">
+        <slot name="otherFilter"></slot>
+      </KGridItem>
+      <KGridItem sizes="9, 9, 9" align="right" class="text-filter">
```

Comment:
Remember that...

| breakpoint | number of columns |
|--|--|
| small | 4 |
| medium | 8 |
| large | 12 |

so I don't think the sizes as given make sense?

Reply:
i don't think it's going to look nice on small or medium, but i'll just split it at 2/2, 3/5, 3/9

Reply:
sometimes for small windows I intentionally split across multiple lines, e.g. 4/4, 4,4, 3/9

Reply:
that's also when I'll set change alignments, so the right column might have `alignments="left, left, right"`

Reply:
I set the props to your suggestions